classes/classes.html

```diff
@@ -49,7 +49,20 @@ <h2 class="title">{{class.name}}</h2>
                 </details>
                 {% if class.lead %}
                 <p>
-                    have questions? <a class="tla-link" href="{{site.baseurl}}/team/{{class.lead.hash}}">contact our curriculum lead {{class.lead.name}}!</a>
+                    have questions? <a class="tla-link" href="{{site.baseurl}}/team/{{class.lead.hash}}">contact our
+                        curriculum lead {{class.lead.name}}!</a>
+                </p>
+                {% endif %}
+                {% if class.leads %}
+                <p>
+                    have questions? contact one of our curriculum leads:
+                <ul>
+                    {% for object in class.leads %}
+                    <li>
+                        <a class="tla-link" href="{{site.baseurl}}/team/{{object.hash}}"> {{object.name}} </a>
+                    </li>
+                    {% endfor %}
+                </ul>
```

Comment:
Instead of creating a line break with `<ul>` and `<li>`, could we just have this be part of the sentence? Ex 

"contact one of our curriculum leads, Eden or Milo".

Reply:
Updated so they're all on the same line. I used the forloop.first check to check whether or not to put the "or" there. Is there a better way to check whether or not to put the "or" there? I tried comparing to the last object in the array but couldn't quite get it to work.

Reply:
`forloop.first` is totally fine, though keep in mind that you want this to be extendable for 3+ school leads. I think a better approach may include:

* adding commas after each person's name
* except for the last person; and, put an "or" in front of their name 
* you can decide if you want to add the comma if there are only two people

To do the second part, you can use `forloop.last`, which works just like `forloop.first`.